### PR TITLE
Pre release 0.21.0

### PR DESCRIPTION
**Pull request by prestist**
https://github.com/coreos/coreos-installer/issues/1425

### CODE REVIEW

**Review by jlebon — COMMENTED** [review]
BTW, we should trim out the empty sections for releases that have gone out already in the release notes. (I see some in the last couple.)

File: docs/release-notes.md [hunk 1]
```diff
@@ -4,7 +4,21 @@ nav_order: 8
...
+Minor changes:
+
+
+Internal changes:
+
+
+Packaging changes:
+
+
+## Upcoming coreos-installer 0.21.0 (2024-02-22)
```
[the remark this answers is not in the document]
Comment (prestist) [reply docs/release-notes.md:21]: wow, sorry yes

**Review by jmarrero — APPROVED** [review]
lgtm

**Comment by prestist** [comment]
Ok I think I have fixed clap now; Will merge once CI passes

**Comment by decathorpe** [comment]
I don't really understand one of the commits from this PR:
https://github.com/coreos/coreos-installer/commit/ee82223b35d352c3de1890f5c9e9394a06eafc73

The commit message is "update dependencies", but the commit does ***not*** update dependencies, it ***restricts*** the versions of clap that can be used. This now broke the package because we updated clap from version 4.4 to 4.5.

**Comment by jlebon** [comment]
@decathorpe Thanks. Follow-up in https://github.com/coreos/coreos-installer/pull/1453.

**Pull request merged**